swift/Samples/ModalContainer/Sources/ModalContainerScreen.swift

```diff
@@ -0,0 +1,50 @@
+import WorkflowUI
+
+
+/// A `ModalContainerScreen` displays a base screen and optionally one or more modals on top of it.
+public struct ModalContainerScreen: Screen {
+
+    /// The base screen to show underneath any modally presented screens.
+    public var baseScreen: AnyScreen
+
+    /// Modally presented screens
+    public var modals: [Modal]
+
+    public init<ScreenType: Screen>(baseScreen: ScreenType, modals: [Modal]) {
+        self.baseScreen = AnyScreen(baseScreen)
+        self.modals = modals
+    }
+
+    public func viewControllerDescription(environment: ViewEnvironment) -> ViewControllerDescription {
+        return ModalContainerViewController.description(for: self, environment: environment)
+    }
+}
+
+
+extension ModalContainerScreen {
+
+    /// Represents a single screen to be displayed modally
+    public struct Modal {
+
+        public enum Style: Hashable {
+            case fullScreen(animated: Bool)
+            case card
```

Comment:
`card` is a Square-specific term for this. Consider something more descriptive of the style (like how it’s `fullScreen` instead of `sheet`)